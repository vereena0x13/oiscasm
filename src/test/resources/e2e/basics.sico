            0 0 start

.define     NL 10

t0:         0
t1:         0

start:      t0 t0 ?+1
            t0 t0 start

            NL