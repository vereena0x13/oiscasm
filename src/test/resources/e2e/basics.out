  0   0   5   0   0   3   3   8   3   3   5  10   3   4   18   1
  2 116 101 115 116   4